docs/reference/gcp/apis.mdx

```diff
@@ -0,0 +1,20 @@
+Nitric APIs are deployed to GCP using [API Gateway](https://cloud.google.com/api-gateway).
+
+## GCP Resources
+
+The following resources are created when deploying APIs to GCP:
+
+- [API Gateway](https://cloud.google.com/api-gateway)
+- [Cloud Run](https://cloud.google.com/run)
+- [GCR](https://cloud.google.com/container-registry)
+
+## Deployment
+
+During deployment Nitric's CLI builds your API's routes, methods and handlers will be built as follows:
```

Comment:
This reads confusingly. Should it be:

During deployment, the Nitric CLI builds your API's routes, methods, and handlers. They are built as follows:

Reply:
We'll have to fix this across all of the other guides since it looks to be the same wording - maybe we can do a bug fix after we release to adjust all guides.